src/main/kotlin/com/fasterxml/jackson/module/kotlin/KotlinModule.kt

```diff
@@ -12,11 +14,21 @@ import java.util.*
 import kotlin.reflect.KClass
 
 private const val metadataFqName = "kotlin.Metadata"
+private const val jsonIncludeAnnotation = "com.fasterxml.jackson.annotation.JsonInclude"
 
 fun Class<*>.isKotlinClass(): Boolean {
     return declaredAnnotations.any { it.annotationClass.java.name == metadataFqName }
 }
 
+fun Class<*>.hasNonNullAnnotations(): Boolean {
+    return declaredAnnotations
```

Comment:
If this reads annotations directly from class definitions, it would break mix-in handling -- all annotation access should really go through `AnnotationXxx` classes (like `AnnotatedClass`).